### PR TITLE
♿Include support for browsers that use contentDocument

### PR DESCRIPTION
**Pull request by Hk-tang**
Include support for browsers that use `.contentDocument` instead of `.contentWindow.document`

The tests all pass locally but not on Travis. No one on the slack has responded to my request for help.

### CODE REVIEW

File: contributing/getting-started-e2e.md [hunk 1]
```diff
@@ -410,7 +410,7 @@ gulp prettify --local_changes --fix
 
 Before sending your code changes for review, please make sure that all of the affected tests are passing. You may be expected to add/modify some tests as well.
 
-Note: You can automatically run critical checks before `git push` by enabling our pre-push hook. To do so, run `./build-system/enable-git-pre-push.sh`.
```
Comment (powerivq) [inline contributing/getting-started-e2e.md]: Thanks for that!

File: extensions/amp-a4a/0.1/template-renderer.js [hunk 2]
```diff
@@ -64,16 +64,19 @@ export class TemplateRenderer extends Renderer {
       }
       const templateHelper = getAmpAdTemplateHelper(context.win);
       return templateHelper
-        .render(data, iframe.contentWindow.document.body)
+        .render(
+          data,
+          (iframe.contentDocument || iframe.contentWindow.document).body
```
Comment (powerivq) [inline extensions/amp-a4a/0.1/template-renderer.js]: Maybe extracting this to a function? We would save some bytes and this seems to be a common usage.

Comment (Hk-tang) [reply extensions/amp-a4a/0.1/template-renderer.js]: Would you like the function to be in a different file than the change here?

File: extensions/amp-a4a/0.1/template-renderer.js [hunk 3]
```diff
@@ -40,6 +40,17 @@ export class TemplateRenderer extends Renderer {
     super();
   }
 
+  /**
+   * Retrieve the content document depending on browser support
+   *
+   * @param {*} iframe
+   *   The iframe to retrieve the document of
+   * @return {*}
```
Comment (Hk-tang) [inline extensions/amp-a4a/0.1/template-renderer.js]: I was not able to determine the type of the param and return of the function

**Comment by powerivq** [comment]
Look like we need more rebasing..

**Pull request merged**